### PR TITLE
Dockerize new

### PR DESCRIPTION
**Pull request by Andrew-EP**
https://jira.gtforge.com/browse/GETT-14886

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
-ADD templates/ /etc/tyk/templates/
+RUN apk add --update git vim bash && rm -rf /var/cache/apk/*
 
-EXPOSE 8080
+RUN echo 'alias ll="ls -la"' >> ~/.bashrc
 
-ENTRYPOINT ["/bin/tyk"]
-CMD []
+ARG BUNDLE_GITHUB__COM
+RUN echo -e "machine github.com\n  login ${BUNDLE_GITHUB__COM}" >> ~/.netrc
```
Comment (cohental) [inline Dockerfile]: why do you need it? I think you need to remove the github bundle. the container does need to access github to run.

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,42 @@
...
+ENV TYKLANG ""
+
+ENV TYKLISTENPORT 8080
+ENV TYKSECRET 352d20ee67be67f6340b4c0605b044b7
+ENV TYK_PATH /opt/tyk-gateway/
+
+LABEL Description="Tyk Gateway docker image" Vendor="Tyk" Version=$TYKVERSION
+
+RUN mkdir -p /go/src/github.com/gtforge/${APP_NAME}
+RUN echo -e "machine github.com\n  login ${BUNDLE_GITHUB__COM}" >> ~/.netrc
```
Comment (cohental) [inline Dockerfile]: same as above

File: Dockerfile [hunk 3]
```diff
@@ -1,11 +1,42 @@
...
+
+RUN echo -e '' > ~/.netrc
+
+ADD . /go/src/github.com/gtforge/${APP_NAME}
+WORKDIR /go/src/github.com/gtforge/${APP_NAME}
+RUN go get && go build -v -i
```
Comment (cohental) [inline Dockerfile]: please delete the source files from the container after the "go build" command. 
They are not needed at runtime.

File: coprocess_grpc.go [hunk 4]
```diff
@@ -4,14 +4,14 @@
 package main
 
 import (
+	"errors"
 	"net"
 	"net/url"
 	"time"
-	"errors"
```
Comment (cohental) [inline coprocess_grpc.go]: why change the import order?

**Comment by Andrew-EP** [comment]
@cohental please check again

**Pull request merged**